packages/wrangler/src/cfetch/internal.ts

```diff
@@ -242,17 +242,16 @@ export async function fetchDashboardScript(
 	if (usesModules) {
 		// Response from edge contains generic "name = worker.js" for dashboard created scripts
 		const form = await response.formData();
-		const entries = Array.from(form.entries());
-		if (entries.length > 1)
-			throw new RangeError("Expected only one entry in multipart response");
-		const [_, file] = entries[0];
+		const files = Array.from(form.entries())
+			.filter(([_, file]) => file instanceof File)
+			.map(([_, file]) => file as File);
 
-		if (file instanceof File) {
-			return await file.text();
-		}
-
-		return file ?? "";
+		return files;
 	} else {
-		return response.text();
+		const contents = await response.text();
+		const filename = response.headers.get("cf-entrypoint") ?? "index.js";
```

Comment:
A service worker syntax worker won't have an entry point (sorry, I think I gave you a different answer in person!)

Reply:
Ok so shall I hardcode 'index.js' instead?

Reply:
I think we've generally tended to use `worker.js` for service workers on the dashboard, but `index.js` probably fits more with Wrangler, so yeah

Reply:
I kept `index.js` to match the existing wrangler functionality and the existing tests